src/styles/less/LoginContainer.less

```diff
@@ -15,59 +15,59 @@
     font-size: 1.5rem;
     font-weight: 600;
     line-height: 2.7rem;
-    color: #47525D;
+    color: #47525d;
     text-align: center;
   }
   label {
     font-size: 1rem;
     font-weight: 700;
     margin-left: 7px;
     margin-bottom: 30px;
-    color: #323F4B;
+    color: #323f4b;
   }
   .o-form-fieldset-container {
     margin-top: 32px;
-    height: 180px;
     display: flex;
     flex-direction: column;
     justify-content: space-between;
-    align-items: center;
     .custom-checkbox {
       display: none;
     }
   }
+
   #okta-signin-username,
   #okta-signin-password,
   #account-recovery-username {
-    width: 375px;
+    width: 314px;
     padding-left: 8px;
-    border: 1px solid #C8D4E1;
+    border: 1px solid #c8d4e1;
     border-radius: 8px;
-    padding: 2.3% 0;
+    padding: 0;
+    margin: 0;
   }
   #okta-signin-username:focus,
   #okta-signin-password:focus,
   #account-recovery-username:focus {
-    outline:none;
-    border: 1px solid #7B8794;
-  }
-  a.email-button {
-    height: 3rem;
-    width: 375px;
-    border-radius: 25px;
-    justify-content: center;
-    cursor: pointer;
-    background: #8CE2E0;
-    border: none;
-    color: #2D2A2B;
-    font-family: 'mulish', sans-serif;
-    font-weight: 700;
-    font-size: 1rem;
-    letter-spacing: 1.25px;
-    text-transform: uppercase;
-    display: flex;
-    align-items: center;
+    outline: none;
+    border: 1px solid #7b8794;
   }
+  // a.registration-link {
```

Comment:
Wondering if the code thats commented out is the registration styling that is in progress or should this be deleted?

Reply:
@ianpaulfo I have removed the commented code from LoginContainer.less now, it wasn't needed.

Reply:
@ianpaulfo @tinomen I've added a brief description to the pull request